setup.py

```diff
@@ -76,7 +76,7 @@ def run_tests(self):
         "parse",
         "parse_type",
         "py",
-        "pytest>=2.9.0",
+        "pytest>=3.0.0",
```

Comment:
3.3.2

Reply:
Right. But since the test suite was not failing on pytest >=3.0.0, I would say that we can leave the `pytest>=3.0.0` requirement, and if somebody reports an issue, we can definitively bump it to `pytest>=3.3.2`